shared/Views/Shared/Components/CourseDetails/_Qualification.cshtml

```diff
@@ -15,12 +15,14 @@
     </summary>
     <div class="govuk-details__text">
       <p class="govuk-body">
-        Qualified teacher status (QTS) allows you to teach in state schools in England.
+        QTS (qualified teacher status) allows you to teach in state schools in England.
       </p>
       <p class="govuk-body">
-        If you want to teach in the rest of the UK or internationally, consider a course that gives you the
-        qualification &lsquo;postgraduate certificate in education with qualified teacher status&rsquo; instead.
-       </p>
+        It may also allow you to teach in <a href="https://www.gov.uk/eu-eea">the EU, the EEA and Switzerland</a>. You should check with the institution you’d like to teach in for confirmation. The government has published <a href="https://www.gov.uk/government/publications/eu-exit-no-deal-preparations-for-schools-in-england/eu-exit-no-deal-preparations-for-schools-in-england#preparations-as-an-employer-including-the-recognition-of-teaching-qualifications-for-eea-and-swiss-nationals">guidance about the status of QTS in these countries after the UK leaves the EU</a>.
```

Comment:
> guidance about the status of QTS in these countries after the UK leaves the EU
It's the other way around. It's about the status of non-UK qualifications in the UK.